src/web/lib/components/ThemeColorsEditor/index.js

```diff
@@ -44,34 +49,33 @@ export default class ThemeColorsEditor extends React.Component {
 
     return (
       <div className="theme-colors-editor">
-        <dl className="colors">
+        <ul className="colors">
           {Object.keys(colors).map((name, idx) => {
             const color = colors[name];
             return [
-              <dt
+              <li
                 key={`dt-${idx}`}
-                className={classnames({ selected: selectedColor === name })}
+                className={classnames(name, 'color', { selected: selectedColor === name })}
                 onClick={() => setSelectedColor({ name })}
               >
-                <span style={{ backgroundColor: colorToCSS(color) }} />
-                {colorLabels[name]}
-              </dt>,
-              <dd
-                key={`dd-${idx}`}
-                className={classnames({ selected: selectedColor === name })}
-              >
-                <SketchPicker
-                  color={{ h: color.h, s: color.s, l: color.l, a: color.a * 0.01 }}
-                  disableAlpha={!colorsWithAlpha.includes(name)}
-                  onChangeComplete={({ hsl: { h, s, l, a } }) =>
-                    setColor({ name, h, s: s * 100, l: l * 100, a: a * 100 })}
-                />
-              </dd>
+                <span className="color__swatch"style={{ backgroundColor: colorToCSS(color) }} />
+                <span className="color__label">{colorLabels[name]}</span>
+                <span className="color__picker">
+                  <BlockPicker
```

Comment:
Personal thing, but I don't like this color picker. It's pretty restrictive - no alpha and I can't pick any colors I like.

Reply:
I'm not thrilled about it either, but it had a caret so i'm using it for the time being